src/pybito.cpp

```diff
@@ -381,6 +423,10 @@ PYBIND11_MODULE(bito, m) {
   // FUNCTIONS
   m.def("ratio_gradient_of_height_gradient", &RatioGradientOfHeightGradientEigen,
         "Obtain a ratio gradient from a height gradient.");
+  m.def("log_det_jacobian_of_height_transform", &LogDetJacobianHeightTransform,
+        "Obtain the log determinant jacobian of a height transform.");
+  m.def("log_det_jacobian_determinant", &GradientLogDeterminantJacobian,
```

Comment:
Could we prepend the python function with `gradient_` or use `gradient_log_det_jacobian_of_height_transform` so it is clear that's the gradient of the function `log_det_jacobian_of_height_transform` defined just before.

Reply:
Okay.  Renamed to `gradient_log_det_jacobian_of_height_transform`.